app/Http/Controllers/Api/V1/FeatureFlagController.php

```diff
@@ -0,0 +1,155 @@
+      *    path="/api/v1/feature-flags",
+      *    operationId="define_feature_flags_from_github",
+      *    tags={"Application"},
+      *    summary="Define feature flags from GitHub or request body",
+      *    description="Validates a bearer token, then either defines feature flags from the request body (JSON), or fetches them from GitHub using a configured URL. Defines features using Laravel Pennant.",
+      *    security={{"bearerAuth":{}}},
+      *    @OA\RequestBody(
+      *        required=false,
+      *        @OA\JsonContent(
+      *            type="object",
+      *            example={
+      *                "createDatasets": {
+      *                    "enabled": true
+      *                },
+      *                "upload": {
+      *                    "enabled": false
+      *                },
+      *                "gmi": {
+      *                    "enabled": true,
+      *                    "features": {
+      *                        "auth": {
+      *                            "enabled": true
+      *                        },
+      *                        "no-auth": {
+      *                            "enabled": false
+      *                        }
+      *                    }
+      *                }
+      *            }
+      *        )
+      *    ),
+      *    @OA\Response(
+      *        response=200,
+      *        description="Success",
+      *        @OA\JsonContent(
+      *            @OA\Property(property="message", type="string", example="Feature flags defined successfully.")
+      *        )
+      *    ),
+      *    @OA\Response(
+      *        response=401,
+      *        description="Unauthorized or invalid token",
+      *        @OA\JsonContent(
+      *            @OA\Property(property="message", type="string", example="Unauthorized: Invalid token.")
+      *        )
+      *    ),
+      *    @OA\Response(
+      *        response=422,
+      *        description="Invalid feature flag format",
+      *        @OA\JsonContent(
+      *            @OA\Property(property="message", type="string", example="Invalid feature flag format.")
+      *        )
+      *    ),
+      *    @OA\Response(
+      *        response=500,
+      *        description="Failed to fetch flags",
+      *        @OA\JsonContent(
+      *            @OA\Property(property="message", type="string", example="Failed to fetch feature flags.")
+      *        )
+      *    )
+      * )
+      */
+    public function index(Request $request, FeatureFlagManager $flagManager): JsonResponse
+    {
+        $featureFlagToken = env('FEATURE_FLAG_API_TOKEN');
+        $authHeader = $request->header('Authorization');
+
+        if (!$authHeader || !str_starts_with($authHeader, 'Bearer ')) {
+            return response()->json(['message' => 'Unauthorized: Missing or invalid Authorization header.'], 401);
+        }
+
+        $providedToken = substr($authHeader, 7); // remove "Bearer "
```

Comment:
Just split the string based on space.
check if is bearer first string and take the second part